src/screens/OtherProfile/index.js

```diff
@@ -99,12 +100,6 @@ const OtherProfile = () => {
     setOtherProfileData: setDataMain
   } = useOtherProfileScreenHooks(params?.data?.other_id, params?.data?.username);
 
-  React.useEffect(() => {
-    if (params.data.following === undefined) {
-      setIsCurrentFollow(dataMain.is_me_following_target);
-    }
-  }, [dataMain.is_me_following_target]);
-
   const isSignedMessageEnabled = dataMain.isSignedMessageEnabled ?? true;
   const isAnonimityEnabled = dataMain.isAnonMessageEnabled && isSignedMessageEnabled;
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the useEffect hook that was previously setting the `isCurrentFollowed` state is a good move since this state has been replaced by a function call. It simplifies the component and removes unnecessary re-renders.
```diff
- React.useEffect(() => {
-   if (params.data.following === undefined) {
-     setIsCurrentFollow(dataMain.is_me_following_target);
-   }
- }, [dataMain.is_me_following_target]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->